docs/ecctl-getting-started.asciidoc

```diff
@@ -1,20 +1,19 @@
 [id="{p}-overview"]
 == Overview
 
-NOTE: **You are looking at the documentation for a beta release. Please keep in mind that backwards-incompatible changes might be introduced in future releases.**
+beta[]
 
 {p} is the CLI on steroids for the Elastic Cloud Enterprise API. It wraps
 typical operations commonly needed by ECE operators within a
 single command line tool.
 
 Benefits of {p}:
 
-* Easier to use for ops than using the user console using the RESTful API directly
+* Easier to use than the Cloud UI or using the RESTful API directly
```

Comment:
@nrichers I assume we are using the term Cloud UI across the docs, right? we had a chat with @zanbel yesterday which one to use :) just double-checking

Reply:
@ppapapetrou76 this is how we define references to the user console for ECE: `:console:       Cloud UI`. So "Cloud UI" is what we use in the ECE docs officially and always have. This is different from SaaS--at some point we probably want to consolidate names, but for now I would suggest using the ECE one.